src/herder/TxSetFrame.cpp

```diff
@@ -659,12 +637,22 @@ TxSetFrame::checkValid(Application& app, uint64_t lowerBoundCloseTimeOffset,
 #endif
     }
 
-    if (this->size(lcl.header) > lcl.header.maxTxSetSize)
+    if (this->size(lcl.header, Phase::CLASSIC) > lcl.header.maxTxSetSize)
     {
-        CLOG_DEBUG(Herder, "Got bad txSet: too many txs {} > {}",
-                   this->size(lcl.header), lcl.header.maxTxSetSize);
+        CLOG_DEBUG(Herder, "Got bad txSet: too many classic txs {} > {}",
+                   this->size(lcl.header, Phase::CLASSIC),
+                   lcl.header.maxTxSetSize);
         return false;
     }
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+    if (this->size(lcl.header, Phase::SOROBAN) > SOROBAN_TX_LIMIT_PER_LEDGER)
```

Comment:
classic->Soroban in the debug log

Also, 2 seems too low (though probably doesn't really matter since we're going to update this soon.

Reply:
yeah, the number doesn't matter, this variable will go away as soon as merge my other PR anyway, I just wanted to have a placeholder, so we don't forget to have this check.